helm/README.md

```diff
@@ -21,40 +21,47 @@ Clone this repository and change directories into `nim-deploy/helm`. The followi
 
-## Setting up your helm values file
+## Setting up your helm values
 
 All available helm values can be discoved by running the `helm` command after downloading the repo.
 
 ```bash
 helm show values nim-llm/
 ```
 
-To start a new value file for a particular cluster, run
+You only need to add values to a file that you want to change from defaults. The values options and defaults are detailed in the chart's [readme](nim-llm/README.md). The `image.repository` and `image.tag` options are especially important if you do not want to deploy the default image for this chart.
+
+To run llama3-8b-instruct using the NIM 1.0.0 release, you need to change very little and don't even truly need a values file. For an example, see [Launching a default NIM with minimal values](#Launching-a-default-NIM-with-minimal-values)
+
+## Create a namespace
+
+For the rest of this guide, we will use the namespace `nim`.
 
 ```bash
-helm show values nim-llm/ > custom-values.yaml
+kubectl create namespace nim
 ```
 
-and edit for the specific configuration. While all values may require editting, pay particular attention to the image.repository which specifies which model the NIM will run. 
-
-The chart requires certain [Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/) to be configured in the cluster.
+## Launching a default NIM with minimal values
 
-* NGC container downloads require an image pull secret (in this case named `registry-secret`).
-* NGC model downloads require an NGC API key (a secret `ngc-api` that has the value stored in a key named `NGC_CLI_API_KEY`).
+`llama3-8b-instruct` at the chart's appVersion (currently 1.0.0) can be launched in a default configuration while only setting the NGC API key and persistence in one line with no extra files. Using Persistent volumes is still recommended to ensure permissions are set correctly and the container runtime filesystem isn't filled by downloading models.
 
-These secrets can be created with the following command:
 ```bash
-kubectl create secret docker-registry registry-secret --docker-server=nvcr.io --docker-username='$oauthtoken' --docker-password=$NGC_CLI_API_KEY
-
-kubectl create secret generic ngc-api --from-literal=NGC_CLI_API_KEY=$NGC_CLI_API_KEY
+helm --namespace nim install my-nim nim-llm/ --set model.ngcAPIKey=$NGC_CLI_API_KEY --set persistence.enabled=true
 ```
 
-### Values
+## Using a custom values file
 
 When deploying NIMs there are several values that can be customized to control factors such as scaling, metrics collection, resource use, and most general AI model selection.
 
-Here is an example using meta/llama-3-8b-instruct.
+Here is an example using meta/llama-3-8b-instruct. Where you have a pre-existing secret for the NGC API key, using persistent value stores to maintain the model cache and a specific non-default image pull secret. This also disables the StatefulSet setup so that it deploys a Deployment object instead.
```

Comment:
```suggestion
The following example uses meta/llama-3-8b-instruct with an existing secret as the value of the NGC API key, sets the persistent value stores to maintain the model cache and a specific non-default image pull secret, and disables the StatefulSet setup so that it deploys a Deployment object instead.
```